pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1157,6 +1187,32 @@ func (n *NetworkProvider) reconcileRDSVpcConfiguration(ctx context.Context, priv
 	return nil
 }
 
+//this function removes tags and reads them to an rds subnet group
+func (n *NetworkProvider) updateSubnetGroupTags(foundSubnet *rds.DBSubnetGroup, clusterID string) error {
+	_, err := n.RdsApi.RemoveTagsFromResource(&rds.RemoveTagsFromResourceInput{
+		ResourceName:aws.String(*foundSubnet.DBSubnetGroupArn),
+		TagKeys: []*string{
+			aws.String("cluster"),
+		},
+	})
+	if err != nil {
+		return errorUtil.Wrap(err, "error updating db subnet group tags")
+	}
+	_, err = n.RdsApi.AddTagsToResource(&rds.AddTagsToResourceInput{
+		ResourceName: aws.String(*foundSubnet.DBSubnetGroupArn),
+		Tags: []*rds.Tag{
+			{
+				Key:   aws.String("cluster"),
+				Value: aws.String(clusterID),
```

Comment:
Might be better to pass in `[]*rds.Tag` to keep this function a bit more generic.